_pages/en/it-picture-medium-term.md

```diff
@@ -30,7 +30,34 @@ In the following sections, we're trying to take the same end state and create "v
 - Cloud First
   - Projects by default must use the cloud. Justification required if the project is not intending to use the cloud 
 - (TODO: What does EAs role look like in 5 years? Sources and references [CDS or 18f, for example])
-- All new projects must deliver a functional product at the end of each quarter, at the absolute latest.
+
+### Greenfield Project Quarterly Delivery 
+
+All new projects must deliver a functional product at the end of each quarter, at the absolute latest. 
+
+In modern IT literature organizations should be aiming to deliver software into production within minutes. In [*War And Peace And IT*](https://itrevolution.com/book/war-and-peace-and-it/) by [Mark Schwartz](https://itrevolution.com/faculty/mark-schwartz/), ex-CIO of US Citizenship and Immigration Services and currently a Strategist for Amazon Web Services, he writes that business or mission needs should be met in hours or days. It is in this spirit that IITB within 5 years needs to be able to deploy solutions to production quarterly, at the very least, by 2025. 
+
+In the [DORA State of DevOps](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=3&cad=rja&uact=8&ved=2ahUKEwiGzKvWsoDkAhVOTt8KHVTvDBoQFjACegQIAhAC&url=http%3A%2F%2Fcloudplatformonline.com%2Frs%2F248-TPC-286%2Fimages%2FDORA-State%2520of%2520DevOps.pdf&usg=AOvVaw3Igoh1u4hGjIr7p9jV1_Ae) report on page 14 they define categories of performance as follows 
+
+![alt text](../../assets/images/aspectOfSoftwareDeliveryPerformance.PNG)
+
+Of respondents 
+
+- 7% were Elite Performers
+- 41% were High Performers
+- 37% were Medium Performers
+- 15% were Low Performers 
+
+Given that low performers are able to deploy between once per week and once per month, within 5 years IITB will must be able to deploy into production within 4 months within 5 years for all new projects (including those replacing legacy systems). 
```

Comment:
reword:
within 5 years IITB will must be able to deploy into production within 4 months within 5 years for all new projects 
to
within 5 years IITB will must be able to deploy into production within 4 months for all new projects

Reply:
Made the recommended changes